lua/gitlinker/git.lua

```diff
@@ -255,6 +280,12 @@ function M.get_repo_data(remote)
   if not repo or vim.tbl_isempty(repo) then
     vim.notify(table.concat(errs), vim.log.levels.Error)
   end
+
+  local branch = get_remote_branch(remote)
+  if branch then
+    repo.rev = branch
+  end
```

Comment:
No need for conditionality here, always set it, even if `nil`, it will be checked later anyways :wink: 

```suggestion
  repo.branch = get_remote_branch(remote)
```